environ/project.clj

```diff
@@ -1,7 +1,18 @@
-(defproject environ "1.1.0"
+(defproject environ "1.1.1"
   :description "Library for accessing environment variables"
   :url "https://github.com/weavejester/environ"
   :scm {:dir ".."}
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
-  :dependencies [[org.clojure/clojure "1.5.1"]])
+  :dependencies [[org.clojure/clojure "1.8.0"]]
```

Comment:
Is the requirement for 1.8 a problem here? And is it possible to avoid it for the clj version?